nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/filters/FacetFilter.java

```diff
@@ -20,32 +20,42 @@
 
 import java.util.List;
 import java.util.Map;
-
-import org.nuxeo.ecm.core.api.DocumentModel;
 import org.nuxeo.ai.pipes.streams.Initializable;
+import org.nuxeo.ecm.core.api.DocumentModel;
 
 /**
  * A Document Facet filter.
  * Check that none match the excluded facets and ANY match the included facets
  */
 public class FacetFilter implements Filter.DocumentFilter, Initializable {
 
-    protected List<String> includedFacets;
-    protected List<String> excludedFacets;
+    protected List<String> facets;
+
+    protected boolean excluded;
 
     @Override
     public void init(Map<String, String> options) {
-        includedFacets = propsList(options.get("includedFacets"));
-        excludedFacets = propsList(options.getOrDefault("excludedFacets", "Folderish"));
+        facets = propsList(options.get("includedFacets"));
+        List<String> excludedFacets = propsList(options.get("excludedFacets"));
+
+        if (excludedFacets != null && !excludedFacets.isEmpty()) {
```

Comment:
excludedFacets takes priority over includedFacets, in the case the two are present. 
maybe add this to javadoc